codegen/src/generator/rest_json.rs

```diff
@@ -116,6 +117,30 @@ impl GenerateProtocol for RestJsonGenerator {
     }
 }
 
+// Used to print the enum value rather than the status code and the canonical reason.
+// For codegen purposes, leaving existing StatusCode Display trait implementation intact.
+// StatusCode::Ok.to_string() prints "200 OK"
+// StatusCode::Ok.enum_as_string() prints "StatusCode::Ok"
+trait CodegenString {
+    fn enum_as_string(&self) -> String;
+}
+impl CodegenString for StatusCode {
+    fn enum_as_string(&self) -> String {
+        format!("StatusCode::{:?}", self)
+    }
+}
+
+fn http_code_to_status_code(code: Option<i32>) -> String {
+    match code {
+        Some(actual_code) => {
+            StatusCode::from_u16(actual_code as u16).enum_as_string()
+        }
+        // Some service definitions such as elastictranscoder don't specify
```

Comment:
Added this comment since elastictranscoder [sometimes specifies return codes](https://github.com/boto/botocore/blob/develop/botocore/data/elastictranscoder/2012-09-25/service-2.json#L111) and [sometimes doesn't](https://github.com/boto/botocore/blob/develop/botocore/data/elastictranscoder/2012-09-25/service-2.json#L127).